keylime/registrar_client.py

```diff
@@ -64,7 +68,20 @@ def getData(registrar_ip, registrar_port, agent_id, tls_context):
             logger.critical("Error: did not receive port from Registrar Server.")
             return None
 
-        return response_body["results"]
+        r = response_body["results"]
+        res: tenant.RegistrarData = {
+            "aik_tpm": r["aik_tpm"],
+            "regcount": r["regcount"],
+            "ek_tpm": r["ek_tpm"],
+            "ip": r["ip"],
+            "port": r["port"],
+            "mtls_cert": r.get("mtls_cert"),
```

Comment:
This line here in tenant.py makes me think that "mtls_cert" must be there but can be None:

https://github.com/keylime/keylime/blob/9dae00e519ba5f37cc7b8bfbebdb2ad6953f03bc/keylime/tenant.py#L951

However, the code block this is in is surrounded with a catcher for an `Exception` that could theoretically catch access to an unavailable dict key.